src/context/status.ts

```diff
@@ -1,4 +1,4 @@
-import * as statuses from 'statuses/codes.json'
+import statuses from 'statuses/codes.json'
```

Comment:
Importing it as `* from X` resulted into the `statuses` being an object with the `{ default }` key. 

Getting the value as `statuses[statusCode]` failed, since it was in fact `statuses.default[statusCode]`.